### PR TITLE
1527 dev move ontouom units to om

### LINKED ISSUE #576: Put additional units into OM, not TWA
We have added numerous units here:
https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_Ontology/ontology/ontouom

This is not good practice. Instead, we should add them directly to this repository:
https://github.com/cambridge-cares/OM

We can/should then submit pull requests to the original repository that it was forked from.

### PR DESCRIPTION
**Pull request by YiKaiTsai**

### CODE REVIEW

**Review by sm453 — COMMENTED** [review]
Looks good to me. While we're on it, given the changes to `MassOfSubstanceConcentration`, would you be able to make the analogous changes to `AmountOfSubstanceConcentration` (for consistency)? I would hope that that's only a small change...

File: om-2.0.rdf [hunk 1]
```diff
@@ -39440,8 +39726,16 @@
...
+  <owl:Class rdf:about="&om;AmountOfMoneyPerArea">
+    <om:commonlyHasUnit rdf:resource="&om;poundSterlingPerSquareMetre"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;AmountOfMoneyPerEnergy">
+    <om:commonlyHasUnit rdf:resource="&om;euroPerMegawattHour"/>
+  </owl:Class>
+
+  <owl:Class rdf:about="&om;AmountOfMoneyPerTimeInterval">
+    <om:commonlyHasUnit rdf:resource="&om;euroPerHour"/>
```
Comment (markushofmeister) [inline om-2.0.rdf:39738]: One could argue whether those `commonlyHasUnit` relationships are required or the normal `Unit` declarations suffice (but not necessay!)

**Review by markushofmeister — COMMENTED** [review]
Looks good! Just two clarification questions:
1) Normally all OM concepts include a dimension definition. We used to have included them for most of our concepts, e.g. `MassOfSubstanceConcentration`. Currently, we do not have those for all `AmountOf Money...` concepts (and likely some more). Shall we be consistent and include them?
2) Are you sure you have included all unit definitions from the [ontouom repo](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_Ontology/ontology/ontouom)? I just checked for `metreAOD` and yes, the unit itself is listed as available option; however, it is not really defined in OM (i.e. symbol, label, etc.) or I have missed it

File: om-2.0.rdf (hunk 1 above)
Comment (YiKaiTsai) [reply om-2.0.rdf:39738]: I don't think the ```commonlyHasUnit``` relationships are necessary. My understanding is that they were added to be consistent with OM, where most quantities have both ```commonlyHasUnit``` relationship and the normal ```Unit``` declaration.

Comment (YiKaiTsai) [reply om-2.0.rdf:39738]: I missed adding in the definition for ```metreAOD``` and the other water level height units, will add them in.

**Review by sm453 — APPROVED** [review]

File: om-2.0.rdf (hunk 1 above)
Comment (markushofmeister) [reply om-2.0.rdf:39738]: Thanks, what about `litrePerDay`, `litrePerSecond`, and `nanogramPerCubicMetre`?

**Comment by markushofmeister** [comment]
> 1. Normally all OM concepts include a dimension definition. We used to have included them for most of our concepts, e.g. `MassOfSubstanceConcentration`. Currently, we do not have those for all `AmountOf Money...` concepts (and likely some more).

So, we skip including the dimension definitions? Strictly speaking, I believe this shall not be done, but do not have strong opinion

**Comment by YiKaiTsai** [comment]
> So, we skip including the dimension definitions? Strictly speaking, I believe this shall not be done, but do not have strong opinion

I am fine with including the dimension definitions. 

@sm453 What's your opinion on adding in the dimension definition for ```AmountOfMoney``` concepts?

**Comment by sm453** [comment]
> @sm453 What's your opinion on adding in the dimension definition for `AmountOfMoney` concepts?

Just to be clear, we're talking about all the units, e.g. `om:UnitedStatesDollar`, to receive hasDimension statements, right? The classes themselves don't seem to have dimensions.

If you can easily do it without too much trouble, then it would of course be nice to have. But as I said previously, we cannot solve all problems in OM that are not of our own making (and this one definitely isn't). If you feel the urge to fix this one, by all means. Just be careful about opening further cans of worms...

**Comment by YiKaiTsai** [comment]
> Just to be clear, we're talking about all the units, e.g. `om:UnitedStatesDollar`, to receive hasDimension statements, right? The classes themselves don't seem to have dimensions.
> 
> If you can easily do it without too much trouble, then it would of course be nice to have. But as I said previously, we cannot solve all problems in OM that are not of our own making (and this one definitely isn't). If you feel the urge to fix this one, by all means. Just be careful about opening further cans of worms...

Thanks for the input.

Considering that there won't be just our own ```AmountOfMoney``` concepts that will be modified, I am opting to leave them as they are now.

**Comment by YiKaiTsai** [comment]
If there are no more desired changes, I will merge the branch.

**Comment by sm453** [comment]
> If there are no more desired changes, I will merge the branch.

Fine from my side.

**Pull request merged**